lib/deep_cover/cli.rb

```diff
@@ -15,7 +15,7 @@ class CLI < Thor
     check_default_type!
 
     # Every top-level commands are defined in a different file, which add their method to this class
-    DeepCover.require_relative_dir 'cli'
+    DeepCover.require_relative_dir 'cli_impl'
```

Comment:
How about `cli_impl` => `cli/commands` instead?

Reply:
We'll talk about it later lol